.docker/monolog.yaml

```diff
@@ -0,0 +1,48 @@
+when@prod:
+    monolog:
+        channels: ["deprecation"]
+        handlers:
+            main:
+                type: fingers_crossed
+                action_level: error
+                handler: nested
+                excluded_http_codes: [403, 404]
+            nested:
+                type: stream
+                level: info
+                path: php://stderr
+            console:
+                type: console
+                process_psr_3_messages: false
+                channels: ["!event", "!doctrine"]
+            deprecation:
+                type: stream
+                channels: ["deprecation"]
+                path: php://stderr
```

Comment:
```suggestion
#            deprecation:
#                type: stream
#                channels: ["deprecation"]
#                path: php://stderr
```

I would currently deactivate deprecation logs, they are full of unfixable Doctrine message.